dev_docs/tutorials/data/search.mdx

```diff
@@ -0,0 +1,490 @@
+    // handle result
+  }, 
+});
+```
+
+##### Async Search and Custom Async Search Strategies
+
+The open source default search strategy (`ES_SEARCH_STRATEGY`), run searches synchronously, keeping an open connection to Elasticsearch the whole time the query executes. The duration of those queries is restricted by the `elasticsearch.requestTimeout` setting in `kibana.yml`, which is 30 seconds, by default.
+
+This synchronous execution works great in most cases. However, with the introduction of features, such as `Data Tiers` and `Runtime Fields`, the need to allow running slower queries, where holding an open connection might be inefficient, has increased. In 7.7, `Elasticsearch` have introduced the [async_search API](https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html), allowing a query to run for longer, without having to keep an open connection. Instead, the initial search request returns an ID, identifying the search running in `Elasticsearch`. This ID can then be used to retrieve, cancel or manage the search result.
+
+
+
+The [async_search API](https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html) is what drives more advanced `Kibana` `search` features, such as `Partial Results` and `Search Sessions`. Therefore, Running with a Basic+ license, the default search strategy of `Kibana` is automatically set to the **async** default search strategy (`ENHANCED_ES_SEARCH_STRATEGY`), empowering kibana to run longer queries, with an **optional** duration restriction defined by the UI setting (`search:timeout`).
+
+If you are implementing your own async custom search strategy, make sure to implement `cancel` and `extend` as shown in the example bellow: 
+
+```ts
+// ./myPlugin/server/myEnhancedStrategy.ts
+export const myEnhancedSearchStrategyProvider = (
+  data: PluginStart
+): ISearchStrategy<IMyStrategyRequest, IMyStrategyResponse> => {
+  // Get the default search strategy
+  const ese = data.search.getSearchStrategy(ENHANCED_ES_SEARCH_STRATEGY);
+  return {
+    search: (request, options, deps) => {
+      // search will be called multiple times, 
+      // be sure your response formatting is capable of handling partial results, as well as the final result.
+      return formatResponse(ese.search(request, options, deps));
+    },
+    cancel: async (id, options, deps) => {
+      // call the cancel method of the async strategy you are using or implement your own cancellation function.
+      await ese.cancel(id, options, deps);
+    },
+    extend: async (id, keepAlive, options, deps) => {
+      // async search results are not stored indefinitely. By default, they expire after 7 days (or as defined by xpack.data_enhanced.search.sessions.defaultExpiration setting in kibana.yml).
+      // call the extend method of the async strategy you are using or implement your own extend function.
+      await ese.extend(id, options, deps);
+    },
+  };
+};
+```
+
+### High Level Search
+
```

Comment:
Would be nice to have a paragraph here with a short description. Why/when would you use a high level search